<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tripPage</class>
 <widget class="QWidget" name="tripPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true"> QPushButton{
        margin-left: 50px; margin-right: 50px;
        background-color: rgb(146, 189, 108);
         border-style: solid;
         border-width: 2px;
         border-color: beige;
         font: bold 14px;
     }
  QPushButton:pressed {
         background-color: rgb(224, 0, 0);     
     }
     QPushButton:hover {
         background-color: rgb(224, 255, 0);
     }

    QPushButton:hover:pressed
    {
        background-color:red;
    }</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true"> QPushButton{
        margin-left: 50px; margin-right: 50px;
        background-color: rgb(146, 189, 108);
         border-style: solid;
         border-width: 2px;
         border-color: beige;
         font: bold 14px;
     }
  QPushButton:pressed {
         background-color: rgb(224, 0, 0);     
     }
     QPushButton:hover {
         background-color: rgb(224, 255, 0);
     }

    QPushButton:hover:pressed
    {
        background-color:red;
    }</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="FoodWindow" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="LabelWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="vboxLabel">
              <property name="spacing">
               <number>5</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="NameWindow" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="cityTitleWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="titleVBox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
